antarest/study/web/raw_studies_blueprint.py

```diff
@@ -59,6 +60,13 @@
     ".json": ("application/json", "utf-8"),
 }
 
+
+class MATRIX_FORMAT(EnumIgnoreCase):
+    JSON = "json"
+    BYTES = "bytes"
```

Comment:
Agreed, "bytes" is not a format, it could be anything inside.
However, output matrices will not return TSV without further handling, so maybe we should just drop this value.

If format is not defined, then we return the raw unformatted bytes.
An alternative is to keep both `formatted` and `format` arguments in the API to make it clearer. If `formatted` is False we return the raw bytes, else we look at format value to choose what to return (json or arrow)

Reply:
I prefer the alternative that I'll implement.